src/base.iced

```diff
@@ -700,6 +715,9 @@ class OuterLink
 
     # For newer clients that push an explicit seq_type value, we push it onto the array here
     arr.push @seq_type if @seq_type?
+
```

Comment:
can you just update the comment about 3 lines above?

Reply:
isn't it still accurate?

Reply:
```
# For backwards-compatibility, if the incoming chainlink doesn't have a
# seq_type, we don't push a null or a defaut value (3), we just leave it as
# a 5-value array
```

If you pass in a 6-value array, we don't add the ignore_if_unsupported for the same reason, is what I meant.